tapdb/interfaces.go

```diff
@@ -49,7 +49,7 @@ type TxOptions interface {
 type BatchedTx[Q any] interface {
 	// ExecTx will execute the passed txBody, operating upon generic
 	// parameter Q (usually a storage interface) in a single transaction.
-	// The set of TxOptions are passed in in order to allow the caller to
+	// The set of TxOptions are passed in order to allow the caller to
```

Comment:
I think this one is actually on purpose. Could perhaps be written as "passed in in-order" to be more clear.

Reply:
"in-order" as in ordered in a specific way? after looking at `TxOptions` there doesn't seem to be any slice or array that holds entities in some order. So perhaps change the phrasing completely and do something like
```suggestion
	// The set of TxOptions are provided in order to allow the caller to
```

Reply:
Ah, no, I misread it myself. So your suggestion is actually better.